### PR TITLE
Initial version.

### PR DESCRIPTION
**Pull request by smarnach**

### CODE REVIEW

File: templates/crafty.service.j2 [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+[Service]
+Type=simple
+ExecStart={{ CRAFTY_HOME }}/venvs/crafty/bin/matterbot
+Restart=always
+RestartSec=10
+WorkingDirectory={{ CRAFTY_HOME }}/crafty
+User=crafty
+Group=crafty
+Envrionment="MATTERMOST_BOT_URL={{ CRAFTY_MATTERMOST_URL }}"
```
Comment (UmanShahzad) [inline templates/crafty.service.j2]: @smarnach Typo -- `Environment`?

Comment (smarnach) [reply templates/crafty.service.j2]: @UmanShahzad Yeah, that's what I meant.  Hey, I'm still working on this. :)

Comment (UmanShahzad) [reply templates/crafty.service.j2]: :p

**Comment by tomaszgy** [comment]
👍

**Pull request merged**